Source/core/resources/countries.json

```diff
@@ -0,0 +1,2058 @@
+{  
+  "BD":{  
+    "name":"Bangladesh",
+    "RenewablePer":"4.11",
+    "currency":"BDT",
+    "cities":{  
+
+    }
+  },
+  "BE":{  
+    "name":"Belgium",
+    "RenewablePer":"13.76",
+    "currency":"EUR",
+    "cities":{  
+      "Brussels":{  
+        "avgCostPT":"2.2"
```

Comment:
Where did you get this data? If possible, please add a comment in the code pointing to the source.